React/Views/RCTWrapperViewController.m

```diff
@@ -92,9 +92,15 @@ - (void)viewWillAppear:(BOOL)animated
     bar.barTintColor = _navItem.barTintColor;
     bar.tintColor = _navItem.tintColor;
     bar.translucent = _navItem.translucent;
-    bar.titleTextAttributes = _navItem.titleTextColor ? @{
-      NSForegroundColorAttributeName: _navItem.titleTextColor
-    } : nil;
+
+    NSMutableDictionary *textAttributes = [[NSMutableDictionary alloc] init];
+    if (_navItem.titleTextColor) {
+      textAttributes[NSForegroundColorAttributeName] = _navItem.titleTextColor;
+    }
+    if (_navItem.titleFont) {
+      textAttributes[NSFontAttributeName] = _navItem.titleFont;
+    }
+    bar.titleTextAttributes = (_navItem.titleTextColor || _navItem.titleFont) ? textAttributes : nil;
```

Comment:
Change the check to `textAttributes.count` so it's easier to add more properties in the future